base-theme/layouts/partials/footer.html

```diff
@@ -1,2 +1,66 @@
-<footer>
+<footer id="footer-container">
+    <div class="row pb-6 mx-0">
+      <div>
+        <a id="open-learning-logo"  href="https://openlearning.mit.edu/" target="_blank">
+          <img src= "/images/mit-ol.png">
+        </a>
+      </div>
+    </div>
+    <div class="row mx-0 justify-content-between">
+      <section class="columns">
+        <div class="about-courseware">
+          <h5>
+            ABOUT MIT OPENCOURSEWARE
+          </h5>
+          <p>
+            MIT OpenCourseWare is an online publication of materials from over 2,500
+            MIT courses, freely sharing knowledge with learners and educators
+            around the world. <a href="/about">Learn more</a>
+          </p>
+        </div>
+        <div class="large-and-above-only row mx-0 align-items-center">
+          <p class="font-weight-bold">PROUD MEMBER OF : <a href="https://www.oeglobal.org/" target="_blank"><img class="oeg-logo" src="/images/oeg_logo.png"></a></p>
+        </div>
+        <div class="copyright large-and-above-only">
+          <p>© 2001–{{ now.Format "2006" }} 	Massachusetts Institute of Technology</p>
+        </div>
+      </section>
+      <section class="columns">
+        <div>
+          <ul>
+            <li><a href="https://accessibility.mit.edu">Accessibility</a></li>
+            <li><a href="https://creativecommons.org/licenses/by-nc-sa/4.0/">Creative Commons License</a></li>
+            <li><a href="https://ocw.mit.edu/terms/">Terms and Conditions</a></li>
```

Comment:
```suggestion
            <li><a href="pages/privacy-and-terms-of-use/">Terms and Conditions</a></li>
```

I'd like to link this to a page managed by OCW Studio. I think this URL will work.

Reply:
I checked RC and its giving me a 404 for this URL ie https://ocwnext-rc.odl.mit.edu/pages/privacy-and-terms-of-use/

Reply:
It's working now. I think the page needed to be published.